### ISSUE TITLE
Partition of Datasets

### ISSUE THREAD

**Issue by FOURIERWANG** [dmlc/dgl#710]
In general, the training set has more data than the test set. Without considering the val set, it's almost seven to three.
But why citation data are the opposite? for example, Cora has 140 train samples，500 val samples，1000 test samples.

**Comment by mufeili** [comment]
Could you please give more context? Which dataset are you referring to?

**Comment by FOURIERWANG** [comment]
I'm using citation_graph data.Cora has 140 train samples，300 val samples，1000 test samples.

**Comment by mufeili** [comment]
We are using the exact setting of [semi-supervised classification with graph convolutional networks](https://arxiv.org/abs/1609.02907). While you may consult the author or the work or the dataset further, I can think of two potential reasons:
1. We are interested in the scenario where there is few labeled data and a lot more unlabeled data. 
2. For semi-supervised learning on graphs, although the objective function is computed on a small subset of the nodes, an n-layer graph neural network implicitly exploits the features of the n-hop neighborhood of the training nodes. So we might actually have more information than your initial thought.

I'm closing the issue for now. If you have some further questions like this, we encourage users to use our [forum](https://discuss.dgl.ai/) for discussion.

**Comment by FOURIERWANG** [comment]
Thanks for your reply.
